src/Form.vue

```diff
@@ -37,6 +37,13 @@
             },
             value: {},
         },
+        mounted() {
+            window.addEventListener('keyup', event => {
+                if (event.keyCode === 13) {
```

Comment:
for clarity, please use the Vue key code constant:
https://vuejs.org/v2/guide/events.html#Key-Codes